### DISCUSSION TITLE
Does the AdvancedMarker support something similar to the anchorPoint on the legacy Marker component?

### DISCUSSION THREAD

**Discussion by TiagoPortfolio** [visgl/react-google-maps#273] [Q&A] [answered]
There are use cases where I want my AdvancedMarker to be rendered anchored to the bottom or center of the point.
I see that the legacy Marker has an [`anchorPoint`](https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerOptions.anchorPoint) prop but I didn't find anything similar on the AdvancedMarker, is this not support yet on the AdvancedMarker?

Cheers!

**Comment by usefulthink** [comment, marked as answer]
You can put any component or html-element inside the `AdvancedMarker` component, and those elements will be rendered such that the top-left corner of the element ends up at the geographic location of the marker. From there you can use regular CSS to reposition the element as needed (e.g. `tranform: translate(-50%, -50%)` to move the anchor-point to the center of the element).
